pom.xml

```diff
@@ -215,6 +215,9 @@
 				<configuration>
```

Comment:
I didn't want to do it as part of this PR because it would've hid the change, but I noticed this file is tab incremented instead of spaces.  I will format it to use spaces before merging this back to develop.